jupyter-config/jupyter_cs3_config.json

```diff
@@ -3,7 +3,7 @@
 	"reva_host": "127.0.0.1:19000",
 	"auth_token_validity": "3600",
 	"endpoint": "/",
-	"home_dir": "/home",
```

Comment:
I wouldn't change the default, unless you think the shares will always need access to the `myshares` folder.. From other changes bellow, this seems to be what you're doing, is that correct?

Reply:
done